classes/Customer.php

```diff
@@ -773,6 +773,45 @@ public static function checkPassword($idCustomer, $passwordHash)
         return Cache::retrieve($cacheId);
     }
 
+    /**
+     * Validate that fields from Customer that are
+     * - required
+     * - boolean
+     * Must be true
+     *
+     * @return array of errors
+     *
+     * @throws PrestaShopException
```

Comment:
You throw nothing Jon Throw :thinking:

Reply:
Copy-paste from similar method in ObjectModel. It might throw, I dont know, it's legacy 😅